src/Serilog.Sinks.MSSqlServer/LoggerConfigurationMSSqlServerExtensions.cs

```diff
@@ -45,14 +46,14 @@ public static class LoggerConfigurationMSSqlServerExtensions
             LogEventLevel restrictedToMinimumLevel = LevelAlias.Minimum,
             int batchPostingLimit = MSSqlServerSink.DefaultBatchPostingLimit,
             TimeSpan? period = null,
-            IFormatProvider formatProvider = null)
+            IFormatProvider formatProvider = null, bool utcTimestamp = true)
```

Comment:
+1 for an opt-in, plus it's convention throughout Serilog that optional Booleans default to false, so helps there too.

Also missing a carriage return to keep the formatting clean in this parameter list.